kivymd/uix/button/button.py

```diff
@@ -1365,20 +1179,20 @@ def on_type(
         self.set_size()
         self.set_font_size()
 
-    def on_touch_down(self, touch):
-        super(MDFloatingActionButton, self).on_touch_down(touch)
-        if self.collide_point(touch.x, touch.y):
-            return True
+    #def on_touch_down(self, touch):
+        #super(MDFloatingActionButton, self).on_touch_down(touch)
+        #if self.collide_point(touch.x, touch.y):
+            #return True
 
-    def on_touch_move(self, touch):
-        super(MDFloatingActionButton, self).on_touch_move(touch)
-        if self.collide_point(touch.x, touch.y):
-            return True
+    #def on_touch_move(self, touch):
+        #super(MDFloatingActionButton, self).on_touch_move(touch)
+        #if self.collide_point(touch.x, touch.y):
+            #return True
```

Comment:
Use PEP8

Reply:
Commented code deleted (should be unnecessary).